### DISCUSSION TITLE
Typing Listbox and RadioGroup Event Handlers

### DISCUSSION THREAD

**Discussion by vhscom** [rgossiaux/svelte-headlessui#50] [General]
Both `Listbox` and `RadioGroup` emit events with a `.details` property for accessing the item selected in a callback handler. What's the best way to strongly type the arguments used in the handlers in TypeScript? Are there any plans to export type definitions for these types of events? Perhaps you had something in mind for #43?

**Comment by rgossiaux** [comment]
My plan is ultimately to deprecate these events entirely and replace them with two-way binding, actually.

Some mild typing improvements to events made their way into the release I just made (1.0.0-beta.9) but they don't help much with the two events you mentioned: bfc68577cc73610fe56c71fb41f1eea05a560921
